### ISSUE TITLE
About when using DatasetPure.build_trainset

### ISSUE THREAD

**Issue by kyorisann** [massquantity/LibRecommender#497]
I am trying to use the data from movie rens.
The algorithm is NFC_neural_collaborative_filtering.
When "DatasetPure.build_trainset" is used to split the data during training,
Is the data of user and item encoded?
If so, when looking at the original data of user and item after the prediction, is it correct to map the original ID from "data_info.user2id"?
Is this correct?

**Comment by massquantity** [comment]
Yes, `DatasetPure.build_trainset` will encode the data. The `predict` function will convert the original user and item internally, so you do not need to map it by yourself.
